ui/lib/kv/addon/components/page/secret/overview.js

```diff
@@ -25,18 +25,28 @@ import { dateFormat } from 'core/helpers/date-format';
  * @param {boolean} canUpdateSecret - permissions to create a new version of a secret
  * @param {model} metadata - Ember data model: 'kv/metadata'
  * @param {string} path - path to request secret data for selected version
- * @param {string} secretState - if a secret has been "destroyed", "deleted" or "created" (still active)
  * @param {object} subkeys - API response from subkeys endpoint, object with "subkeys" and "metadata" keys
  */
 
 export default class KvSecretOverview extends Component {
-  get isActive() {
-    const state = this.args.secretState;
-    return state !== 'destroyed' && state !== 'deleted';
+  get secretState() {
+    if (this.args.metadata) {
+      return this.args.metadata?.currentSecret.state;
```

Comment:
nit: if you're checking for metadata already you don't need the inline ternary
```suggestion
      return this.args.metadata.currentSecret.state;
```